Cognite.Simulator.Utils/Cognite.Simulator.Utils.csproj

```diff
@@ -23,8 +23,10 @@
 
   <ItemGroup>
     <PackageReference Include="Cognite.ExtractorUtils" Version="1.29.0" />
+    <PackageReference Include="Microsoft.Bcl.TimeProvider" Version="10.0.0-preview.1.25080.5" />
```

Comment:
Why do we use the preview version?

Reply:
Good catch, moved this to the last non-preview version.